tests/repl_util.erl

```diff
@@ -195,9 +196,15 @@ wait_until_fullsync_stopped(SourceLeader) ->
                   end).
 
 wait_for_reads(Node, Start, End, Bucket, R) ->
+    wait_for_x(Node, fun() -> rt:systest_read(Node, Start, End, Bucket, R, <<>>, true) end).
+
+wait_for_deletes(Node, Start, End, Bucket, R) ->
+    wait_for_x(Node, fun() -> rt:systest_verify_delete(Node, Start, End, Bucket, R) end).
+
+wait_for_x(Node, Fun) ->
```

Comment:
Maybe "wait_for_empty_result" would be a more appropriate name - rename "Reads" to "Results" as well.